### PR TITLE
Feature/82 easy deployment with heroku

### PR DESCRIPTION
**Pull request by tjarbo**
> **As a** User
> **I want to** host the fmdb in the cloud and deploy it with "one" click
> **because** I am not familiar with docker and my fmdb should run 24/7

## Description
This is issue #82. I have added a entire new wiki article to setup a fmdb with Heroku and MongoDB Atlas.
https://github.com/tjarbo/discord-moodle-bot/wiki/Setup-with-heroku-and-mongodb-atlas

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe):

## Checklist

- [ ] ~Make sure you are requesting to development. Don't request to master branch!~
- [x] I have linked the related issue.
- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  "success_url": "/login",
+  "scripts": {
+    "postdeploy": "npm run build"
+  },
+  "buildpacks": [
+    { "url": "heroku/nodejs" }
+  ],
+  "env": {
+    "ADMIN_ID": {
+      "description": "Discord-Id of the initial administrator. More can be added later vie webinterface."
```
Comment (p-fruck) [inline app.json:23]:
```suggestion
      "description": "Discord-Id of the initial administrator. More can be added later via webinterface."
```

**Review by p-fruck — CHANGES_REQUESTED** [review]
Sieht gut aus, aber einen schwerwiegenden Fehler möchte ich anmerken :P

File: app.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    },
+    "MOODLE_BASE_URL": {
+      "description": "URL of your moodle instance you got from https://fmdb.tjarbo.me"
+    },
+    "MOODLE_FETCH_INTERVAL": {
+      "description": "Defines how often the fmdb looks for updates. Intervall in milliseconds.",
+      "value": "900000"
+    },
+    "MOODLE_REMINDER_TIME_LEFT": {
+      "description": "If the remaining time of a submission is less than the specified time, a notification will be sind. Time in seconds.",
```
Comment (antonplagemann) [inline app.json:58]:
```suggestion
      "description": "If the remaining time of a submission is less than the specified time, a notification will be sent. Time in seconds.",
```

File: app.json [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      "value": "86400"
+    },
+    "MOODLE_TOKEN": {
+      "description": "Token you got from https://fmdb.tjarbo.me ."
+    },
+    "MOODLE_USE_COURSE_SHORTNAME": {
+      "description": "Use the course form or the full course name in Discord messages.",
```
Comment (antonplagemann) [inline app.json:65]: Wäre einheitlicher, außerdem habe ich die Formulierung davor nicht ganz verstanden :-)
```suggestion
      "description": "Whether to use short- or fullname of the courses in the discord message",
```

**Review by antonplagemann — CHANGES_REQUESTED** [review]
Habe auch noch schwerwiegende Fehler gefunden :-D
Sieht gut aus ;-)

**Review by antonplagemann — APPROVED** [review]

**Comment by tjarbo** [comment]
Nooo... GitHub, was kannst du eigentlich 🙄 Bitte nochmal approven @antonplagemann

**Review by antonplagemann — APPROVED** [review]
Und es ist sogar noch schöner geworden! 🎉

**Comment by tjarbo** [comment]
> Und es ist sogar noch schöner geworden! tada

:laughing:

**Pull request merged**